examples/graphql-mutations/graphql_mutations.bal

```diff
@@ -0,0 +1,63 @@
+import ballerina/graphql;
+
+service /graphql on new graphql:Listener(4000) {
+
+    // Define a `Person` object in the service
+    private Person person;
+
+    function init() {
+        // Initialize the `person` value
+        self.person = new("Walter White", 51);
+
+    }
+
+    // A resource function represents a field in root `Query` operation
+    resource function get profile() returns Person {
+
```

Comment:
Do we need above new line ? I would rather prefer without it.

Reply:
We cannot have a GraphQL service without a `Query` object. Therefore, we must have a resource function in order to be a valid GraphQL service.